cmake/modules/FindQwt.cmake

```diff
@@ -35,25 +35,13 @@
 
 SET(QWT_FOUND "NO")
 
-FIND_PATH(QWT_INCLUDE_DIR qwt.h
-  /usr/local/qwt/include
-  /usr/local/include
-  /usr/include/qwt6
-  /usr/include/qwt-qt4
-  /usr/include/qwt
-  /usr/include
-  /opt/local/include/qwt #macports path
-  $ENV{QWT_DIR}/include
-  $ENV{QWT_DIR}/src
-  $ENV{QWTDIR}/include
-  $ENV{QWTDIR}/src
-  $ENV{QWT_ROOT}/include
-  $ENV{QWT_ROOT}/src
-  $ENV{QWTROOT}/include
-  $ENV{QWTROOT}/src
+find_path ( QWT_INCLUDE_DIR
```

Comment:
Minor issue - try to keep the same style for CMake calls to avoid mixing styles in files.

Reply:
fixed